app/views/employers/commitments.html.erb

```diff
@@ -17,72 +17,96 @@
   <div class="section one">
     <% unless user_signed_in? && current_user.is_employer? %>
       <div class="row">
-          <div class="small-12 columns">
-          <p>Employers every day are committing to hire Veterans &mdash; you should too! Check out the list below to see what employers have already made a commitment and then <%= link_to "create or update your profile", new_user_session_path %> to signal your commitment.</p>
+        <div class="small-12 medium-8 columns">
+          <div class="primary">
+            <p>Veterans and military family members have consistently proven themselves to be the most reliable, independent, and hardworking team members. Additionally, Veterans have soft skills such as problem solving and creativity that can benefit a company’s bottom line. By signing up and <a href="/employment/commitments">making a commitment</a>, your company will stand out to candidates eager to enter the civilian workforce.</p>
```

Comment:
We're already on the commitments page, so having a link here does not seem helpful (will just be a link to the page you're already on)